crawler/file_processing.py

```diff
@@ -639,10 +683,16 @@ def filtered_row(self, row, line_number) -> Dict[str, str]:
                 self.log_adding_default_lab_id(row, line_number)
 
         # filter out any unexpected columns
+        # get values for required headers
```

Comment:
```suggestion
        # get names of required column headers
```

Reply:
We're getting the values from those column fields in the file row, and adding them into the modified row dict.
I guess it's k,v pairs really

Reply:
oh I see, I misread it as just referring to the get_required_headers method, but it's referring to what the whole block does

Reply:
I'll make it clearer then

Reply:
Done